main.go

```diff
@@ -280,17 +280,33 @@ func computeBalanceBuildBackend(network Network) (backend.Backend, error) {
 	return b, err
 }
 
-// pick a default server for each network if none provided
-// TODO: default server should be localhost for Btcd backend.
-func getServer(network Network, addr *net.TCPAddr) (string, string) {
-	if addr != nil {
-		return addr.IP.String(), strconv.Itoa(addr.Port)
+// pick a default server for electrum or localhost for btcd
+func getServer(network Network, backend, addr string) (string, string) {
+	if addr != "" {
+		parts := strings.Split(addr, ":")
```

Comment:
there's https://golang.org/pkg/net/#SplitHostPort for that, unless we're doing something non-standard

Reply:
I want it to work with ipv6.